### DISCUSSION TITLE
Printed Circuit Board Files

### DISCUSSION THREAD

**Discussion by neotil1** [srozum/film_camera_tester#3] [Q&A] [answered]
I'm amazed at how much time and effort you've put into this project. Thank you for the great documentation!

I would like to build my own, however I wasn't able to find any files to oder my own PCBs on here. Are those only available if the tester is purchased as a kit?

**Comment by srozum** [comment, marked as answer]
Uploaded in /assets/pcb folder

**Reply by neotil1** [reply]
Awesome, thank you so much! I've already ordered all the parts and am excited to build my own when they finally arrive in about a month or so.

**Reply by HostoPhoto** [reply]
Hi, 
Where did you order your pcb ?

**Reply by neotil1** [reply]
> Hi, Where did you order your pcb ?

I ordered mine on JLCPCB today. Do you happen to live in the EU? The minimum order is 5pcs of each board so I have a bunch of spares :)

**Reply by HostoPhoto** [reply]
I'm in France. I'll be happy to get one of each. Shoot me an email : contact@hostophoto.fr

**Reply by ItsMeAubey** [reply]
What source do you intend to run on them? I can't seem to find it in the github project

**Reply by neotil1** [reply]
> What source do you intend to run on them? I can't seem to find it in the github project

What do you mean by that? The firmware is located here if that's what you meant: https://github.com/srozum/film_camera_tester/releases

**Reply by srozum** [reply]
Nice, I am glad to see you collaborating. Keep going, I will be glad to help if needed.

**Reply by Pitichinaccio** [reply]
Hi neotil1,

I am also interested in the PCBs. Could you please contact me : phillip.maus@gmx.de
Many thanks!
